app/src/main/java/org/simple/clinic/summary/nextappointment/NextAppointmentUpdate.kt

```diff
@@ -3,12 +3,14 @@ package org.simple.clinic.summary.nextappointment
 import com.spotify.mobius.Next
 import com.spotify.mobius.Next.next
 import com.spotify.mobius.Update
+import org.simple.clinic.mobius.dispatch
 
 class NextAppointmentUpdate : Update<NextAppointmentModel, NextAppointmentEvent, NextAppointmentEffect> {
 
   override fun update(model: NextAppointmentModel, event: NextAppointmentEvent): Next<NextAppointmentModel, NextAppointmentEffect> {
     return when (event) {
       is NextAppointmentPatientProfileLoaded -> next(model.nextAppointmentPatientProfileLoaded(event.nextAppointmentPatientProfile))
+      NextAppointmentActionButtonClicked -> dispatch(OpenScheduleAppointmentSheet(model.patientUuid))
```

Comment:
There are a few conditions in the PRD where we don't have to open `ScheduleAppointmentSheet`, are we doing this somewhere in the upcoming PR's?

Reply:
Those cases are for when done or back is clicked. We don't have to worry about it for this feature. It's [tracked](https://app.shortcut.com/simpledotorg/story/6431/change-appointment-bottom-sheet-display-logic) in a different story.